<ui version="4.0" >
 <class>LogIntegratorFrontEndClass</class>
 <widget class="QMainWindow" name="LogIntegratorFrontEndClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ログ統合ソフト</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title" >
     <string>ロット番号で指定</string>
    </property>
    <widget class="QPushButton" name="pushButton_Run" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>70</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text" >
      <string>処理開始</string>
     </property>
     <property name="shortcut" >
      <string>F5</string>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_LotNo" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>30</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength" >
      <number>36767</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>160</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title" >
     <string>日付で指定</string>
    </property>
    <widget class="QPushButton" name="pushButton_DateRun" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>70</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text" >
      <string>処理開始</string>
     </property>
     <property name="shortcut" >
      <string>F5</string>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_Date" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>30</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbClose" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>閉じる</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_S" >
    <property name="title" >
     <string>システム(&amp;S)</string>
    </property>
    <addaction name="actionShowSettingDialog" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menu_S" />
  </widget>
  <action name="actionShowSettingDialog" >
   <property name="text" >
    <string>初期設定(&amp;I)</string>
   </property>
   <property name="iconText" >
    <string>初期設定(I)</string>
   </property>
   <property name="toolTip" >
    <string>ログのあるHDや出力先フォルダを設定します</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>終了(&amp;Q)</string>
   </property>
   <property name="toolTip" >
    <string>ログ統合ソフトを終了します</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>lineEdit_LotNo</tabstop>
  <tabstop>pushButton_Run</tabstop>
  <tabstop>dateEdit_Date</tabstop>
  <tabstop>pushButton_DateRun</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
